app/controllers/user/listings_controller.rb

```diff
@@ -1,18 +1,40 @@
 class User::ListingsController < ApplicationController
   def show
     @listing = Listing.find(params[:id])
+    @listing_images = @listing.listing_images.all
+    @remaining_images = @listing_images[1..-1]
```

Comment:
We could probably just call these in the view off of the listing rather than make variables out of them